tests/slack_sdk_async/web/test_async_web_client_http_retry.py

```diff
@@ -88,3 +84,30 @@ async def test_fatal_error(self):
         client.retry_handlers.append(FatalErrorRetryHandler())
         # The auto-retry should work here
         await client.auth_test()
+
+    @async_test
+    async def test_retries(self):
```

Comment:
Moved from tests/slack_sdk_async/web/test_web_client_http_retry.py to reduce confusion for future maintainers